packages/loader/driver-utils/src/toBuffer.ts

```diff
@@ -0,0 +1,15 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IsoBuffer } from "@fluidframework/common-utils";
+
+/**
+ * Convert base64 or utf8 string to array buffer
+ */
+export function toBuffer(input: string, encoding: string): ArrayBufferLike {
+    console.log(IsoBuffer.from(input,encoding));
+    console.log(IsoBuffer.from(input, encoding).buffer);
+    return IsoBuffer.from(input, encoding).buffer;
```

Comment:
the .buffer method/property is returning a way oversized string. idk what the best method to use here to return ArrayBufferLike

Reply:
I'm not sure what you mean. Code looks correct to me, but please remove all console.log statements before merge